dub.sdl

```diff
@@ -4,6 +4,7 @@ authors "Sönke Ludwig" "Martin Nowak" "Matthias Dondorff" "Sebastian Wilzbach"
 	"more than 80 contributors total"
 copyright "Copyright © 2012-2016 rejectedsoftware e.K., Copyright © 2012-2014 Matthias Dondorff"
 license "MIT"
+dflags "-preview=dip1000"
```

Comment:
Can you move this change to the CI config for now where we know when we're using dmd-nightly?

Reply:
Seconded, this will otherwise also appear in projects that use DUB as a library.

Reply:
Done.